src/main/java/org/broadinstitute/hellbender/utils/read/CigarBuilder.java

```diff
@@ -123,4 +134,17 @@ private void advanceSectionAndValidateCigarOrder(CigarOperator operator) {
             }
         }
     }
+
+    public int getLeadingDeletionBasesRemoved() {
+        return leadingDeletionBasesRemoved;
+    }
+
+    public int getTrailingDeletionBasesRemoved() {
+        return trailingDeletionBasesRemoved + trailingDeletionBasesRemovedInMake;
+    }
+
+    public Triple<Cigar, Integer, Integer> makeAndRecordDeletionsRemoved() {
```

Comment:
Document these. Also I would make this return an object rather than a triple.

Reply:
done